.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules
-public/*
```

Comment:
No longer need to gitignore this - built output goes to `build/public` now, which makes way more sense.

Any files in `public/` get copied to `build/public` as part of the backend esbuild process.